# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qemu/zoom_to_fit.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- qemu-5.2.0/ui/gtk.c.vanilla	2020-12-09 15:29:52.358392832 +0100
+++ qemu-5.2.0/ui/gtk.c	2020-12-09 15:29:57.526392838 +0100
@@ -1983,7 +1983,7 @@
                               QemuConsole *con, int idx,
                               GSList *group, GtkWidget *view_menu)
 {
-    bool zoom_to_fit = false;
+    bool zoom_to_fit = true;
 
     vc->label = qemu_console_get_label(con);
     vc->s = s;
